app/jobs/update_job.rb

```diff
@@ -13,7 +13,8 @@ class UpdateJob < ApplicationJob
   # @param [BackgroundJobResult] background_job_result
   # rubocop:disable Metrics/AbcSize
   # rubocop:disable Metrics/MethodLength
-  def perform(model_params:, signed_ids:, background_job_result:)
+  # rubocop:disable Metrics/PerceivedComplexity
+  def perform(model_params:, signed_ids:, background_job_result:, start_workflow: true)
```

Comment:
this changes the current default behavior.  wasn't sure which way to go, but since this is consistent with `IngestJob`, and since it's the behavior we seemed to expect by default, i went this way.  happy to change, of course.